src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
+  const object = {};
+
+  for (const item of splitArray) {
+    if (item.trim().length > 0) {
+      const resultItemSplit = item.split(':');
+      const key = resultItemSplit[0].trim();
+      const value = resultItemSplit[1].trim();
+
+      object[key] = value;
+    }
+  }
```

Comment:
Instead of using a for...of loop, consider using array methods like 'map' and 'filter' to transform the array of style declarations into an object. This would adhere to the checklist requirement to avoid loops and use iteration methods.